src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileAccessCategory.scala

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage.fileitem
+
+import nl.knaw.dans.common.lang.dataset.AccessCategory
+import nl.knaw.dans.common.lang.dataset.AccessCategory._
+
+object FileAccessCategory extends Enumeration {
+  type FileAccessCategory = Value
+  val ANONYMOUS, KNOWN, RESTRICTED_REQUEST, RESTRICTED_GROUP, NONE = Value
+
```

Comment:
You could, temporarily, but AFAIK accessibility and visibility are not the same!

If the keys (eg enum members) are the same, you could maybe rewrite this enumeration to an abstract class (sealed?) with `accessibility` and `visibility` texts and make instances (case classes or objects) for each accessibility type. See for a similar thing the [KeywordMapping in the License Creator](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/EASY-1039/src/main/scala/nl/knaw/dans/easy/license/KeywordMapping.scala) or [this assignment solution from the scala workshops](https://github.com/DANS-KNAW/course-scala/blob/master/src/main/scala/workshop1/assignments/UnitConversion.sc).